### PR TITLE
Check if all attributes set correctly in Analyzer

### PR DESCRIPTION
**Pull request by zurk**
Minor improvement

### CODE REVIEW

**Comment by zurk** [comment]
Also, I mention that `description` attribute is missing. We have it in we have `description` field in all our analyzers. Should we add it here?

File: lookout/core/analyzer.py [hunk 1]
```diff
@@ -160,6 +160,10 @@ def __init__(self, model: AnalyzerModel, url: str, config: Mapping[str, Any]):
         :param url: The analyzed project's Git remote.
         :param config: Configuration of the analyzer of unspecified structure.
         """
+        for attr_name in ("version", "model_type", "name", "vendor"):
+            if getattr(self, attr_name) is None:
```
Comment (vmarkovtsev) [inline lookout/core/analyzer.py]: `getattr(self, attr_name)` raises an AttributeError if no attribute is present. OK?

Comment (zurk) [reply lookout/core/analyzer.py]: We have all these arguments is set to `None` (https://github.com/src-d/lookout-sdk-ml/pull/79/files#diff-1d5639c92b3eba213af016871b0e26d2R153). Just join 4 separate checks for them into the loop.

**Comment by vmarkovtsev** [comment]
@k Can you fix the CI?

**Comment by zurk** [comment]
yes. done

**Pull request merged**